2-mandatory/4-people-i-know.js

```diff
@@ -403,15 +419,26 @@ This time, I only want the full names of the people who can multitask
 
 let colleaguesWhoCanMultitask = [];
 
+for (let friend of friends) {
+  for (colleague of friend.colleagues) {
+    if (colleague.skills.includes("Multi-tasking"))
+      colleaguesWhoCanMultitask.push(colleague.name);
+  }
+}
+
```

Comment:
Good - there are few different ways of doing each of these last 2 questions, so if you have time, review the solution or your fellow trainees' solutions next week.